### PR TITLE
lots of apc,cisco, dell additions

### PR DESCRIPTION
**Pull request by ITJamie**
lots of apc,cisco, dell additions

### CODE REVIEW

**Comment by ITJamie** [comment]
finally got tests passing

File: device-types/Dell/PowerEdge_R620xl_solidfire.yml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+manufacturer: Dell
+model: SolidFire OEMR R620 XL
+slug: dell_solidfire_omer_r620xl
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_R620xl_solidfire.yml:11]: We are no longer including USB ports as "console" unless they are specifically console ports

File: device-types/Dell/PowerEdge_R620xl_solidfire.yml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_R620xl_solidfire.yml:22]: We are no longer including USB ports as "console" unless they are specifically console ports

File: device-types/Dell/PowerEdge_R620xl_solidfire.yml [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+power-ports:
+  - name: Power 1
+    type: iec-60320-c14
+  - name: Power 2
+    type: iec-60320-c14
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_R620xl_solidfire.yml:27]: Might be better off including a module bay instead

File: device-types/Dell/PowerEdge_r200.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
+  - name: Rear Keyboard ps2
+    type: mini-din-8
+  - name: Rear Mouse ps2
+    type: mini-din-8
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r200.yml:24]: See comment about USB ports, it is equally as relevant for DIN ports

File: device-types/Dell/PowerEdge_r230.yml [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r230.yml:20]: See comment about USB ports

File: device-types/Dell/PowerEdge_r310.yml [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r310.yml:20]: See comment about USB ports

File: device-types/Dell/PowerEdge_r410.yml [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+manufacturer: Dell
+model: PowerEdge R410
+slug: dell_poweredge_r410
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r410.yml:11]: See comment about USB ports

File: device-types/Dell/PowerEdge_r410.yml [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r410.yml:22]: See comment about USB ports

File: device-types/Dell/PowerEdge_r415.yml [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+manufacturer: Dell
+model: PowerEdge R415
+slug: dell_poweredge_r415
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r415.yml:11]: See comment about USB ports

File: device-types/Dell/PowerEdge_r415.yml [hunk 10]
```diff
@@ -0,0 +1,35 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r415.yml:22]: See comment about USB ports

File: device-types/Dell/PowerEdge_r420.yml [hunk 11]
```diff
@@ -0,0 +1,35 @@
...
+manufacturer: Dell
+model: PowerEdge R420
+slug: dell_poweredge_r420
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r420.yml:11]: See comment about USB ports

File: device-types/Dell/PowerEdge_r420.yml [hunk 12]
```diff
@@ -0,0 +1,35 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r420.yml:22]: See comment about USB ports

File: device-types/Dell/PowerEdge_r430.yml [hunk 13]
```diff
@@ -0,0 +1,39 @@
...
+manufacturer: Dell
+model: PowerEdge R430
+slug: dell_poweredge_r430
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r430.yml:11]: See comment about USB ports

File: device-types/Dell/PowerEdge_r430.yml [hunk 14]
```diff
@@ -0,0 +1,39 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r430.yml:22]: See comment about USB ports

File: device-types/Dell/PowerEdge_r440.yml [hunk 15]
```diff
@@ -0,0 +1,39 @@
...
+manufacturer: Dell
+model: PowerEdge R440
+slug: dell_poweredge_r440
+u_height: 1
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r440.yml:11]: See comment about USB ports

File: device-types/Dell/PowerEdge_r440.yml [hunk 16]
```diff
@@ -0,0 +1,39 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r440.yml:22]: See comment about USB ports

File: device-types/Dell/PowerEdge_r510.yml [hunk 17]
```diff
@@ -0,0 +1,35 @@
...
+manufacturer: Dell
+model: PowerEdge R510
+slug: dell_poweredge_r510
+u_height: 2
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r510.yml:11]: See comment about USB ports

File: device-types/Dell/PowerEdge_r510.yml [hunk 18]
```diff
@@ -0,0 +1,35 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r510.yml:22]: See comment about USB ports

File: device-types/Dell/PowerEdge_r520.yml [hunk 19]
```diff
@@ -0,0 +1,35 @@
...
+manufacturer: Dell
+model: PowerEdge R520
+slug: dell_poweredge_r520
+u_height: 2
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r520.yml:11]: See comment about USB ports

File: device-types/Dell/PowerEdge_r520.yml [hunk 20]
```diff
@@ -0,0 +1,35 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r520.yml:22]: See comment about USB ports

File: device-types/Dell/PowerEdge_r540.yml [hunk 21]
```diff
@@ -0,0 +1,35 @@
...
+manufacturer: Dell
+model: PowerEdge R540
+slug: dell_poweredge_r540
+u_height: 2
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r540.yml:11]: See comment about USB ports

File: device-types/Dell/PowerEdge_r540.yml [hunk 22]
```diff
@@ -0,0 +1,35 @@
...
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
+power-ports:
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r540.yml:23]: See comment about USB ports

File: device-types/Dell/PowerEdge_r710.yml [hunk 23]
```diff
@@ -0,0 +1,39 @@
...
+manufacturer: Dell
+model: PowerEdge R710
+slug: dell_poweredge_r710
+u_height: 2
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r710.yml:11]: See comment about USB ports

File: device-types/Dell/PowerEdge_r710.yml [hunk 24]
```diff
@@ -0,0 +1,39 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r710.yml:22]: See comment about USB ports

File: device-types/Dell/PowerEdge_r715.yml [hunk 25]
```diff
@@ -0,0 +1,39 @@
...
+manufacturer: Dell
+model: PowerEdge R715
+slug: dell_poweredge_r715
+u_height: 2
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r715.yml:11]: See comment about USB ports

File: device-types/Dell/PowerEdge_r715.yml [hunk 26]
```diff
@@ -0,0 +1,39 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r715.yml:22]: See comment about USB ports

File: device-types/Dell/PowerEdge_r730xd.yml [hunk 27]
```diff
@@ -0,0 +1,39 @@
...
+manufacturer: Dell
+model: PowerEdge R730xd
+slug: dell_poweredge_r730xd
+u_height: 2
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r730xd.yml:11]: See comment about USB ports

File: device-types/Dell/PowerEdge_r730xd.yml [hunk 28]
```diff
@@ -0,0 +1,39 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r730xd.yml:22]: See comment about USB ports

File: device-types/Dell/PowerEdge_r820.yml [hunk 29]
```diff
@@ -0,0 +1,39 @@
...
+manufacturer: Dell
+model: PowerEdge R820
+slug: dell_poweredge_r820
+u_height: 2
+is_full_depth: true
+console-ports:
+  - name: Serial
+    type: de-9
+  - name: USB
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r820.yml:11]: See comment about USB ports

File: device-types/Dell/PowerEdge_r820.yml [hunk 30]
```diff
@@ -0,0 +1,39 @@
...
+    type: other
+console-server-ports:
+  - name: Front USB1
+    type: usb-a
+  - name: Front USB2
+    type: usb-a
+  - name: Rear USB1
+    type: usb-a
+  - name: Rear USB2
+    type: usb-a
```
Comment (DanSheps) [inline device-types/Dell/PowerEdge_r820.yml:22]: See comment about USB ports

**Review by DanSheps — CHANGES_REQUESTED** [review]
In addition to the comments, this conflicts with #784

**Comment by ITJamie** [comment]
Cool will work on this later. Will that other dell pr be merged soon?

**Pull request merged**